experimental/traffic-portal/src/app/shared/tp-header/tp-header.service.ts

```diff
@@ -22,15 +21,31 @@ import {TpHeaderComponent} from "src/app/shared/tp-header/tp-header.component";
 	providedIn: "root"
 })
 export class TpHeaderService {
-	private header?: TpHeaderComponent;
+	private readonly headerTitle: ReplaySubject<string>;
+	private readonly headerHidden: ReplaySubject<boolean>;
+
+	constructor() {
+		this.headerTitle = new ReplaySubject(1);
+		this.headerHidden = new ReplaySubject(1);
+		this.headerHidden.next(true);
+	}
+
+	/**
+	 * Gets the header title.
+	 *
+	 * @returns Subject containing the header title.
+	 */
+	public getTitle(): ReplaySubject<string> {
+		return this.headerTitle;
+	}
```

Comment:
why not just make the property `public` instead of adding an identity function wrapper?

Reply:
also applies to `getHidden`, obviously.